ext/prism/extconf.rb

```diff
@@ -40,17 +40,22 @@ def generate_templates
   end
 end
 
+require "rbconfig"
+
 # Runs `make` in the root directory of the project. Note that this is the
 # `Makefile` for the overall project, not the `Makefile` that is being generated
 # by this script.`
 def make(target)
   Dir.chdir(File.expand_path("../..", __dir__)) do
-    system(RUBY_PLATFORM.include?("openbsd") ? "gmake" : "make", target, exception: true)
+    system(
+      RbConfig::CONFIG.slice(*%w[SOEXT CPPFLAGS CFLAGS CC AR ARFLAGS MAKEDIRS RMALL]), # env
```

Comment:
Should we extract this operation out so it’s not re-`#slice`d on every call?

Reply:
That's okay, it's not a big deal

Reply:
Actually, on hindsight, should we send `RbConfig::MAKEFILE_CONFIG` (unexpanded `Makefile` includes) over rather than `RbConfig::CONFIG` (expanded `Makefile` values)?

Reply:
I assume it wouldn't make a difference right?

Reply:
The difference would be if the `RbConfig::MAKEFILE_CONFIG` values were expanded from variables that
* our hand-written `Makefile` does not cover / cover differently, or
* interpret users’ env vars on a finer level